src/Microsoft.DotNet.Interactive.ExtensionLab/MsSqlKernelConnection.cs

```diff
@@ -55,8 +90,8 @@ private async Task InitializeDbContextAsync(MsSqlConnectionOptions options, Kern
             context.Display($"Scaffolding a `DbContext` and initializing an instance of it called `{options.KernelName}` in the C# kernel.", "text/markdown");
 
             var submission1 = @$"
-#r ""nuget:Microsoft.EntityFrameworkCore.Design""
-#r ""nuget:Microsoft.EntityFrameworkCore.SqlServer""
+#r ""nuget:Microsoft.EntityFrameworkCore.Design,3.1.8""
+#r ""nuget:Microsoft.EntityFrameworkCore.SqlServer,3.1.8""
```

Comment:
This can move to a .NET 5 version now.

Reply:
Fixed and unit test passes

Reply:
I take that back... the scaffold unit test is failing so I'm investigating this.

Reply:
Try version 5.0.1.

Reply:
5.0.1 didn't help. Here's the exception I'm seeing from submitting the scaffolding code. 3.1.8 works, but 3.1.9 and above fail.

 ```
System.MissingMethodException: Method not found: 'Microsoft.Extensions.DependencyInjection.ServiceProvider Microsoft.Extensions.DependencyInjection.ServiceCollectionContainerBuilderExtensions.BuildServiceProvider(Microsoft.Extensions.DependencyInjection.IServiceCollection)
```

Reply:
Narrowed this down a bit. The unit tests are loading assemblies from `dotnet-interactive/artifacts/bin/Microsoft.DotNet.Interactive.ExtensionLab.Tests/Debug/net5.0` which contain the wrong Microsoft.Extensions.DependencyInjection.dll 3.1.9 assembly rather than using the nuget store where the other EF 5.0 binaries are loaded.

Reply:
Two package dependencies are causing this (so far):
1. Microsoft.ApplicationInsights.AspNetCore is pulling in all kinds of Microsoft.Extension.* dependencies, including the Microsoft.Extensions.DependencyInjection.
2. ICSharpCode.Decompiler is pulling in an outdated version of Humanizer, which conflicts with the version used by EF 5.0, resulting in the same Missing Method exception.

For 2, splitting the SQL kernel into its own assembly would resolve this long term. I need read up a bit more about how to address 1 in the .net core world. Let me know if you have ideas.

Reply:
Would moving some of these dependencies to newer version help? If we're not yet on Microsoft.Extensions.DependencyInjection 5.0.1, we probably should be.

Reply:
I fixed this by declaring explicit nuget dependencies on EF and the Humanizer.Core packages for the ExtensionLab csproj. The existing code had runtime dependencies on EF. Since that dependency was not declared, the build tooling would binplace the minimum package version detected for other projects. Be making them explicit, the newer package versions are binplaced. Unit EF scaffolding tests now pass after some minor changes to the code injected into the kernel.